examples/simple-page/bemhtml-templates.js

```diff
@@ -0,0 +1,118 @@
+module.exports = function() {
+  // 1. Common HTML elements — doctype, html, head, meta, title, body
+  block('page').wrap()(function() {
+    var ctx = this.ctx;
+    var data = ctx.data;
+
+    return [
+      '<!DOCTYPE html>',
+      {
+        tag: 'html',
+        attrs: { lang: 'en-US' },
+        content: [
+          {
+            tag: 'head',
+            content: [
+              { tag: 'meta', attrs: { charset: 'utf-8' } },
+              { tag: 'title', content: ctx.title }
+            ]
+          },
+          {
+            tag: 'body',
+            mix: { block: 'vcard' }, // see vcard template below
+            content: ctx
+          }
+        ]
+      }
+    ];
+  });
+
+  // vcard encapsulates all about person schema (HTML class and attributes)
+  block('vcard')(
+    bem()(false),
```

Comment:
I think that an example of `cls` is not so important to create strange things like `bem()(false) + cls()()`